content/logs/java.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Java log Collection
+kind: documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+beta: true
+---
+
+Java logs are quite complex to handle, mainly because of stack traces. These stack traces are split into multiple lines which makes them difficult to associate to the original log event: :
```

Comment:
This could be simplified to:
"Java logs can be complex due to their multiline format, which makes them difficult to associate to the original log event:"

Reply:
:+1: